composer.json

```diff
@@ -17,7 +17,7 @@
   "require-dev": {
     "laminas/laminas-cache": "^3.0.x-dev",
     "laminas/laminas-cache-storage-adapter-benchmark": "^1.0",
-    "laminas/laminas-cache-storage-adapter-test": "2.0.x-dev",
+    "laminas/laminas-cache-storage-adapter-test": "2.3.x",
```

Comment:
@boesing can we use semver versions here, or do we need to stick to `-dev` versions by design?